api/src/pcapi/sandboxes/scripts/creators/industrial/__init__.py

```diff
@@ -14,6 +14,7 @@
 from pcapi.sandboxes.scripts.creators.industrial.create_industrial_event_offers import *
 from pcapi.sandboxes.scripts.creators.industrial.create_industrial_event_products import *
 from pcapi.sandboxes.scripts.creators.industrial.create_industrial_event_stocks import *
+from pcapi.sandboxes.scripts.creators.industrial.create_industrial_incidents import create_industrial_incidents
```

Comment:
```suggestion
from pcapi.sandboxes.scripts.creators.industrial.create_industrial_incidents import *
```

Reply:
Mouais. On n'est pas obligé de reproduire les cochonneries de "import *", même par souci d'homogénéité. ;)